From cd76ac258cd92a6425cb68f95a76d6abdce5395a Mon Sep 17 00:00:00 2001
From: Thomas Backlund <tmb@mageia.org>
Date: Sat, 4 Apr 2020 23:29:43 +0300
Subject: [PATCH] exfat: add missing MODULE_ALIAS_FS()
Git-commit: cd76ac258cd92a6425cb68f95a76d6abdce5395a
Patch-mainline: v5.7-rc3
References: bsc#1182989

This adds the necessary MODULE_ALIAS_FS() to exfat so the module gets
automatically loaded when an exfat filesystem is mounted.

Signed-off-by: Thomas Backlund <tmb@mageia.org>
Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 fs/exfat/super.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fs/exfat/super.c b/fs/exfat/super.c
index 30e914ad17b5..86c875606024 100644
--- a/fs/exfat/super.c
+++ b/fs/exfat/super.c
@@ -718,6 +718,7 @@ static void __exit exit_exfat_fs(void)
 module_init(init_exfat_fs);
 module_exit(exit_exfat_fs);
 
+MODULE_ALIAS_FS("exfat");
 MODULE_LICENSE("GPL");
 MODULE_DESCRIPTION("exFAT filesystem support");
 MODULE_AUTHOR("Samsung Electronics Co., Ltd.");
-- 
2.26.2

